modules/alb-controller/providers.tf

```diff
@@ -1,11 +1,18 @@
 provider "aws" {
   region = var.region
 
-  profile = coalesce(var.import_profile_name, module.iam_roles.terraform_profile_name)
+  profile = module.iam_roles.profiles_enabled ? coalesce(var.import_profile_name, module.iam_roles.terraform_profile_name) : null
+
+  dynamic "assume_role" {
+    for_each = module.iam_roles.profiles_enabled ? [] : ["role"]
+    content {
+      role_arn = coalesce(var.import_role_arn, module.iam_roles.terraform_role_arn)
+    }
+  }
 }
 
 module "iam_roles" {
-  source  = "../account-map/modules/iam-roles"
+  source  = "../../account-map/modules/iam-roles"
```

Comment:
We should probably move this module into the `eks` directory otherwise this path doesn't make sense

Reply:
But we don't have a precedence for this yet so I'm ok with keeping it where it is for now